guides/Getting Started.md

```diff
@@ -72,6 +72,18 @@ config :friends, Friends.Repo,
 
 This configures how Ecto will connect to our database, called "friends". Specifically, it configures a "repo". More information about [Ecto.Repo can be found in its documentation](https://hexdocs.pm/ecto/Ecto.Repo.html).
 
+It should be noted that some adapter options may be set here. For example, suppose you have a setup that requires unamed prepared statements  The Postgres adapter has an option for this and can be set in your repo config.
+
+```elixir
+config :friends, Friends.Repo,
+  adapter: Ecto.Adapters.Postgres,
+  database: "friends_repo",
+  username: "user",
+  password: "pass",
+  hostname: "localhost"
+  prepare: :unamed
```

Comment:
I'm not sure if this is the best spot, likely it should go in `Ecto.Adapters.Postgres`. Also the correct spelling is `:unnamed`.